CHANGELOG-v2.md

```diff
@@ -157,6 +157,7 @@
   - `verifyJwtWithKey`
   - `wrapJwtInHeader`
   - `convertToNumber`
+  - [analytics] Deprecate the `@sap-cloud-sdk/analytics` package
```

Comment:
```suggestion

- [analytics] Remove the `@sap-cloud-sdk/analytics` package
```

This package is removed from the changelog perspective.
All 1.x.y will be deprecated including analytics.